### PR TITLE
added Devices button, added toml for 'Bitbucket'

### PR DESCRIPTION
**Pull request by mmahmoudian**

### CODE REVIEW

File: data/apps/microsoft.toml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name="Microsoft"
 sessionsURL="https://account.microsoft.com/devices/"
```
Comment (bilelmoussaoui) [inline data/apps/microsoft.toml]: This should be removed?

File: data/apps/google.toml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 name="Google"
 sessionsURL="https://myaccount.google.com/device-activity"
```
Comment (bilelmoussaoui) [inline data/apps/google.toml]: Same

**Comment by mmahmoudian** [comment]
I think now they are resolved

**Comment by bilelmoussaoui** [comment]
Looks good! thanks for your contributions :)

**Pull request merged**